src/extra/AutoUI/index.tsx

```diff
@@ -142,6 +148,16 @@ export const AutoUI = <T extends AutoUIBaseResource<T>>({
 
 	React.useEffect(() => {
 		setSelected([]);
+		if (filters.length) {
+			const { pathname } = window.location;
+			history?.replace?.({
```

Comment:
Do we really need this (if so let's leave a comment on why)?
Isn't the `history.replace` that the PersistentFilters do enough?

Reply:
I needed it at some point while testing my changes, but I guess it's no longer necessary. Removed it